server/services/actions/types.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
     ADD_APPROVAL_REQUEST: 'ADD_APPROVAL_REQUEST',
     EDIT_APPROVAL_REQUEST: 'EDIT_APPROVAL_REQUEST',
     ADD_CASE_APPEAL: 'ADD_CASE_APPEAL',
-    MANAGE_APPEAL_DOCUMENTS: 'MANAGE_APPEAL_DOCUMENTS',
+    ADD_APPEAL_DOCUMENT: 'MANAGE_APPEAL_DOCUMENTS',
```

Comment:
Did you want to change the value here for consistency as well?

Reply:
I have addressed this